response_operations_ui/exceptions/error_codes.py

```diff
@@ -14,15 +14,19 @@ class ErrorCode(Enum):
     API_OIDC_CREDENTIALS_ERROR = "E0006"
     API_TIMEOUT_ERROR = "E0007"
 
+    # HTTP Error Codes
+    NO_RESULTS_FOUND = "404"
```

Comment:
This should probably be `NOT_FOUND` rather than NO_RESULTS_FOUND. As it refers to any requested resource (page/file/rest resource) which is subtly different form the list of CIR metadata being requested in this particular PR.

Reply:
Updated